src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -229,25 +212,41 @@ public ResponseEntity<Void> cutAndPasteNode(@PathVariable("studyUuid") UUID stud
         return ResponseEntity.ok().build();
     }
 
-    @GetMapping(value = "/studies/{studyUuid}/network")
+    @RequestMapping(value = "/studies/{studyUuid}/network", method = RequestMethod.HEAD)
     @Operation(summary = "check study root network existence")
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "204", description = "The network does exists"),
-        @ApiResponse(responseCode = "404", description = "The network doesn't exist - Will try to reimport the network if params is set to true")})
-    public ResponseEntity<Network> getStudyNetwork(@PathVariable("studyUuid") UUID studyUuid,
-                                                   @RequestParam("reimportNetworkIfNotFound") boolean reimportNetworkIfNotFound,
-                                                   @RequestHeader(HEADER_USER_ID) String userId) {
+        @ApiResponse(responseCode = "200", description = "The network does exists"),
```

Comment:
```suggestion
        @ApiResponse(responseCode = "200", description = "The network does exist"),
```
No need an 's' is there is already 'does' I think

Reply:
fixed